source/4.0/guides/sql-how-to/store-data-json-formatting.html.md

```diff
@@ -0,0 +1,51 @@
+---
+chapter: SQL How To
+title: Store data as JSON Strings
```

Comment:
I like the general concept of this guide, but it may beess confusing if we used an example other than JSON so Postgres users can benefit. What about using an XML snippet, or CSV?